### PR TITLE
A bit more friendly language

### PR DESCRIPTION
**Pull request by jmlundberg**
Several small edits, not trying to change the meaning.
- Trying to make the text more inclusive. For example, few can write a completely correct make_unique. "supercalifragilisticexpialidocious" is funny but distracting.
- Trying to turn negative statements into positive ones. Examples.

### CODE REVIEW

File: CppCoreGuidelines.md [hunk 1]
```diff
@@ -120,7 +120,7 @@ Please remember that one purpose of a guideline is to help someone who is less e
 
 Many of the rules are designed to be supported by an analysis tool.
 Violations of rules will be flagged with references (or links) to the relevant rule.
-We do not expect you to memorize all the rules before trying to write code.
+We do not expect programmers to memorize all the rules.
 One way of thinking about these guidelines is as a specification for tools that happens to be readable by humans.
```
Comment (gdr-at-ms) [inline CppCoreGuidelines.md]: The direct style was OK.

File: CppCoreGuidelines.md [hunk 2]
```diff
@@ -347,7 +347,7 @@ Philosophy rules summary:
 
 Philosophical rules are generally not mechanically checkable.
 However, individual rules reflecting these philosophical themes are.
-Without a philosophical basis the more concrete/specific/checkable rules lack rationale.
+Philosophical rules serve as a basis and rationale for the concrete/specific/checkable rules.
```
Comment (gdr-at-ms) [inline CppCoreGuidelines.md]: The original phrasing is better.

File: CppCoreGuidelines.md [hunk 3]
```diff
@@ -8644,6 +8649,10 @@ Look for raw pointers that are targets of `new`, `malloc()`, or functions that m
 
 ### <a name="Res-const"></a>ES.25: Declare objects `const` or `constexpr` unless you want to modify its value later on
 
+##### Note
+
+It's often better to not use pointers at all. [Rr-scoped]
+
 ##### Reason
```
Comment (gdr-at-ms) [inline CppCoreGuidelines.md]: This note seems out of place.

File: CppCoreGuidelines.md [hunk 4]
```diff
@@ -9382,6 +9390,10 @@ The call will most likely be `f(0, 1)` or `f(1, 0)`, but you don't know which. T
 
 Can be detected by a good analyzer.
 
+##### Note
+
+A proposal (P0145R0) relating to this is on its way into C++17. `f(++i, ++i);` remains undefined behaviour.
+
```
Comment (gdr-at-ms) [inline CppCoreGuidelines.md]: This note is not necessary.

Comment (jmlundberg) [inline CppCoreGuidelines.md]: OK.

File: CppCoreGuidelines.md (hunk 3 above)
Comment (jmlundberg) [inline CppCoreGuidelines.md]: Oh yes, sorry. Let's kill that.

File: CppCoreGuidelines.md (hunk 2 above)
Comment (jmlundberg) [inline CppCoreGuidelines.md]: OK. As a non-native English speaker I prefer fewer negations.

File: CppCoreGuidelines.md (hunk 1 above)
Comment (jmlundberg) [inline CppCoreGuidelines.md]: Ok. ".. before trying to write code." felt a bit harsh..

File: CppCoreGuidelines.md (hunk 2 above)
Comment (Pazns) [inline CppCoreGuidelines.md]: The meaning of those two sentences is not the same.
In the new one, "philosophical rules" are both the basis and the rationale.
In the original, they are just the basis.

**Comment by AndrewPardoe** [comment]
Closing based on the discussion. Thank you!

**Comment by jmlundberg** [comment]
Would you like a new pull request without the rejected parts?

**Comment by AndrewPardoe** [comment]
Yes, thanks for noticing that. I erred in closing this. 

Would you please prepare a PR that has just the parts GDR didn't reject?

**Comment by jmlundberg** [comment]
I made an updated PR: https://github.com/isocpp/CppCoreGuidelines/pull/582

**Comment by AndrewPardoe** [comment]
OK, I'll close this in favor of #582. But note GDR has comments on that one : )

**Pull request closed without merging**